staging/src/k8s.io/client-go/discovery/discovery_client.go

```diff
@@ -189,7 +194,6 @@ func (d *DiscoveryClient) ServerGroups() (apiGroupList *metav1.APIGroupList, err
 	// populate the map of hashes for each group version
 	d.hashesMu.Lock()
 	defer d.hashesMu.Unlock()
-	d.hashesByGroupVersion = make(map[string]string)
```

Comment:
i.e. why remove this line and move it up rather than setting the `v1` field here?